src/main/resources/xsd/netex_part_3/part3_fares/netex_typeOfTravelDocument_version.xsd

```diff
@@ -166,6 +166,21 @@ Rail transport, Roads and Road transport
 									<xsd:documentation>ALTERNATIVE NAMES for TRAVEL DOCUMENT.</xsd:documentation>
 								</xsd:annotation>
 							</xsd:element>
+							<xsd:element name="validityScope" type="TravelDocumentValidityScopeEnumeration" minOccurs="0">
+								<xsd:annotation>
+									<xsd:documentation>The validity scope for TRAVEL DOCUMENT</xsd:documentation>
+								</xsd:annotation>
+							</xsd:element>
+							<xsd:element ref="TravelDocumentProvider" minOccurs="0">
+								<xsd:annotation>
+									<xsd:documentation>The provider for the TRAVEL DOCUMENT</xsd:documentation>
+								</xsd:annotation>
+							</xsd:element>
```

Comment:
Hmm konvensjon på store små bokstaver. Ser ut til å være lower case for lister og uppercase ellers.